Holovibes/sources/gui/other/popup_error.cc

```diff
@@ -5,15 +5,30 @@ namespace holovibes::gui
 {
 void show_error_and_exit(const std::string& error_msg, const int exit_value)
 {
-    QMessageBox messageBox;
-    messageBox.critical(nullptr, "Internal Error", error_msg.c_str());
-    exit(exit_value);
+    if (!holovibes::Holovibes::instance().is_cli)
```

Comment:
Maybe add another condition to avoir the message to appear infinitly until the UI crashes. Or put the QMessage in static in order to have a single instance of it

Reply:
I don't understand what you are saying : it makes the program stop so there is no infinitly loop (alexis)